### ISSUE TITLE
Boilerplate and Clis documentation is incorrect (electron-forge section)

### ISSUE THREAD

**Issue by ipoint-pgerhard** [electron/electron#19971]
### Preflight Checklist

* [x] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for an issue that matches the one I want to file, without success.

### Issue Details

Currently the documentation related to electron-forge states
```
Forge comes with ready-to-use templates for popular frameworks like React, Vue, or Angular.
```
however this is not correct.
When opening the link the documentation shows that electron-forge no longer supports the listed frameworks. Instead a single template containing using webpack as a bundler is provided. I would suggest correcting this in the electron docs.

**Comment by ipoint-pgerhard** [comment]
I changed the documentation and created a pull-request
https://github.com/electron/electron/pull/19975

**Comment by ckerr** [comment]
Fixed in #19975.
